### ISSUE TITLE
Patient Clinic IDs

### ISSUE THREAD

**Issue by chris-githu** [chap-uzima/IQCare#65]
Hi @vogoti @mkaburu 
Do we have sites that are using **both** Patient Enrollment ID and Patient Clinic ID to identify patients?
Also sites using Patient Clinic ID or another ID other than Patient Enrollment ID?

`Patient EnrollmentID : 648-100-15282-11604           Patient ClinicID : 873016`

**Comment by mkaburu** [comment]
@chris-githu  Yes there are some sites that use both of them. the Clinic ID is the hospital number while the Patient EnrollmentID is the CCC number. For sites using another number apart from Patient Enrollment ID or Clinic ID am not sure

**Comment by chris-githu** [comment]
@mkaburu 
Thanks.
We'll need to design for this. `(Stress!)`

In the meantime, please find out about other IDs in use in the CCC.
